ts.ts

```
@@ -0,0 +1,43 @@
+
+fetchData().then(data => console.log(data));
+
+function getUserDetails(id: number) {
+  return {
+    id,
+    name: "John Doe",
+    email: "johndoe@example.com"
+  };
+}
+
+let globalCounter = 0;
```

Biome

[error] This let declares a variable that is only assigned once.

'globalCounter' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)